Makefile

```diff
@@ -340,13 +336,9 @@ endif
 # Protobufs #
 #############
 
-protos: $(PROTO_GOS)
-
-# use with care. This signals to make that the proto definitions don't need recompiling.
-touch-protos:
-	for proto in $(PROTO_GOS); do [ -f "./$${proto}" ] && touch "$${proto}" && echo "touched $${proto}"; done
+protos: clean-protos $(PROTO_GOS)
```

Comment:
delete all the compile protos so they will be guaranteed to be built every time this is called.